File: app/src/main/java/com/ducluanxutrieu/quanlynhanvien/Models/Staff.java
package com.ducluanxutrieu.quanlynhanvien.Models;

import java.io.Serializable;

public class Staff implements Serializable {
    private String name;
    private String email;
    private String phone;
    private String position;
    private String uid;
    private boolean admin;
    private String avatarUrl;
    private String password;

    public Staff(String name, String email, String phone, String position, String uid, String avatarUrl, String password, boolean admin) {
        this.name = name;
        this.email = email;
        this.phone = phone;
        this.position = position;
        this.uid = uid;
        this.avatarUrl = avatarUrl;
        this.password = password;
        this.admin = admin;
    }

    public Staff() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public boolean isAdmin() {
        return admin;
    }

    public void setAdmin(boolean admin) {
        this.admin = admin;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public void setAvatarUrl(String avatarUrl) {
        this.avatarUrl = avatarUrl;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

}

File: app/src/main/java/com/ducluanxutrieu/quanlynhanvien/models/Staff.kt
package com.ducluanxutrieu.quanlynhanvien.models

import java.io.Serializable

class Staff : Serializable {
    var name: String? = null
    var email: String? = null
    var phone: String? = null
    var position: String? = null
    var uid: String? = null
    var isAdmin: Boolean = false
    var avatarUrl: String? = null
    var password: String? = null

    constructor(name: String, email: String, phone: String, position: String, uid: String, avatarUrl: String, password: String, admin: Boolean) {
        this.name = name
        this.email = email
        this.phone = phone
        this.position = position
        this.uid = uid
        this.avatarUrl = avatarUrl
        this.password = password
        this.isAdmin = admin
    }

    constructor() {}

}
